test/test_module_init.py

```diff
@@ -0,0 +1,330 @@
+import inspect
```

Comment:
To run the tests add the file name here:

https://github.com/pytorch/pytorch/blob/d4045e9aa173b99d1135b4a64473a0fb630758d9/test/run_test.py#L31

And verify the tests appear in the PR CI output.

Reply:
Thanks!